Fix mini typos in docs (#958)
alan-turing-institute/MLJ.jl: docs/src/evaluating_model_performance.md

[Performance Measures](performance_measures.md).

In addition to hold-out and cross-validation, the user can specify

> their list of train/test pairs of row indices for resampling, or
> define their re-usable resampling strategies.

Reviewer: ```suggestion
an explicit list of train/test pairs of row indices for resampling, or
define new resampling strategies.
```

## User-specified train/test sets

> Users can either provide their list of train/test pairs of row indices for resampling, as in this example:

Reviewer: ```suggestion
Users can either provide an explicit list of train/test pairs of row indices for resampling, as in this example:
```

## Custom resampling strategies

> To define your resampling strategy, make relevant parameters of

Reviewer: ```suggestion
To define a new resampling strategy, make relevant parameters of
```
